lib/generators/rodauth/install_generator.rb

```diff
@@ -71,9 +74,14 @@ def create_mailer
         end
 
         def create_fixtures
-          test_unit_options = ::Rails.application.config.generators.options[:test_unit]
-          if test_unit_options[:fixture] && test_unit_options[:fixture_replacement].nil?
-            template "app/test/fixtures/accounts.yml"
+          test_unit_options = ::Rails.application.config.generators.options[:test_unit]          
+          if test_unit_options[:fixture] && test_unit_options[:fixture_replacement].nil?                                    
+            if defined?(RSpec::Rails)              
+              copy_file "app/test/fixtures/accounts.yml" "app/spec/fixtures/accounts.yml"               
+            else
+              template "app/test/fixtures/accounts.yml" 
```

Comment:
I noticed this only know, but it seems to be copying to `app/test/` instead of `test/`.

Reply:
good find. i have rectified.